lib/cpp/antaris_sdk_environment.cc

```diff
@@ -107,10 +107,19 @@ static void parse_a_conf(char *conf_line, conf_t *out_conf)
 
 static void update_a_conf(char *conf_line)
 {
+    int8_t length = -1;
     conf_t a_conf = {0};
 
     parse_a_conf(conf_line, &a_conf);
 
+    do {
```

Comment:
can we use len = strlen(a_conf.value) to do the same thing ? If len is greater than MAX len, its error.

Reply:
I had revised my comment to use strnlen_s. Can you please check the comment again.